src/otx/algo/visual_prompting/zero_shot_segment_anything.py

```diff
@@ -668,7 +669,8 @@ def __init__(
         )
 
         self.save_outputs = save_outputs
-        self.root_reference_info: Path = Path(root_reference_info)
+        self.reference_info_dir: Path = Path(reference_info_dir)
```

Comment:
Create a JIRA ticket and add a TODO link here about migrating this logic into model checkpoint

Reply:
We agreed to the model checkpoint way afterwards

Reply:
If so, I'll update it in the next PR.
But locally saving them must be maintained because OVModel cannot refer torch checkpoint.
It can be duplicated operations.

Reply:
Last time, we agreed that `OTXVisualPromptingModelExporter` can charge of that responsibility for OVModel.

Reply:
I heard the decision right now from wonju.